File: Logic/dsf.cs
using System;

public class Desafio
{
    public static void Main()
  { 

    float salaryValue = float.Parse(Console.Readline());
    float benefitsValue = float.Parse(Console.Readline());

    float feeValue = 0;
    if (salaryValue >= 0 && salaryValue <= 1100)
    {
        feeValue = 0.05F * salaryValue;
    }
    else if (salaryValue >= 1100.01 && salaryValue <= 2500.00)
    {
        feeValue = 0.10F * salaryValue;
    }
    else 
    {
        feeValue = 0.15F * salaryValue;
    }

    float output = salaryValue - feeValue + benefitsValue;
    Console.Writeline(output.Tostring("0.00"));

  } 
}

File: Logic/dsf.java
import java.util.scanner;

public class desafio {
    
    public static void main(String[] args) {

        Scanner entryReader = new Scanner(System.in);
        float  salaryValue = entryReader.nextfloat();
        float valueBenefits = entryReader.nextfloat();

        float feeValue = 0;
        if (salaryValue >= 0 && salaryValue <= 1100) {
            feeValue = 0.05F * salaryValue;
        
        } else if (salaryValue >= 1100.01 && salaryValue <= 2500.00) {
            feeValue = 0.10F * salaryValue;
        } else {
            salaryValue = 0.15F * salaryValue;
        }

        float output = salaryValue - feeValue + valueBenefits;
        system.out.printIn(String.format("%.2f". output));
    }

}